website/src/components/Submission/DataUploadForm.tsx

```diff
@@ -37,6 +37,9 @@ type DataUploadFormProps = {
 
 const logger = getClientLogger('DataUploadForm');
 
+// This should be the same value as defined in the backend.
+const MAX_UPLOAD_SIZE = '800 MB';
+
```

Comment:
Can you be more specific; is it this property? `spring.servlet.multipart.max-file-size`?

Also, it looks like the PR sets this property to 200MB, so they are not the same.